packages/frontend-api/src/Model/Resolver/Article/ArticleQuery.php

```diff
@@ -50,27 +50,27 @@ public function articleByUuidOrUrlSlugQuery(?string $uuid = null, ?string $urlSl
     }
 
     /**
-     * @return array
+     * @return string
      */
-    public function termsAndConditionsArticleQuery(): array
+    public function termsAndConditionsArticleUrlQuery(): string
     {
-        return $this->getSpecialArticle(Setting::TERMS_AND_CONDITIONS_ARTICLE_ID, 'terms-and-conditions');
+        return $this->getSpecialArticle(Setting::TERMS_AND_CONDITIONS_ARTICLE_ID, 'terms-and-conditions')['slug'][0];
```

Comment:
can we use `['mainSlug']` instead of `['slug'][0]`?

Reply:
Done, thanks :) I didn't know it's there. Where is the structure defined?

Reply:
in the `project-base/app/src/Resources/definition/blog_article` and `project-base/app/src/Resources/definition/article` are definitions for the elasticsearch mapping. 
Useful only when the article is loaded from the elasticsearch ofc.